certifier/tests/test_certifier_models.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from mixer.backend.django import mixer
+
+from certifier.models import *
+
+pytestmark = pytest.mark.django_db
+
+class TestRunReconstruction:
```

Comment:
These tests are not incorrect, but a little silly. If we keep it like this we generate fake coverage, which will lead to untested code. Is there any actual logic that we can test?